lib/models/user.rb

```diff
@@ -4,4 +4,6 @@ class User
   include DataMapper::Resource
 
   property :id, Serial, key: true
-end
\ No newline at end of file
+  property :name, String, length: 100
```

Comment:
You don't have to set the `String` length here. Strings by default have 255 characters maximum. There's not a good reason to limit that further in this case.